<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BPMOptionsPage</class>
 <widget class="QWidget" name="BPMOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>273</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="bpm_options">
     <property name="title">
      <string>BPM Analyze Parameters:</string>
     </property>
     <widget class="QLabel" name="slider_default">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>80</y>
        <width>43</width>
        <height>19</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Most accurate"/>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider_parameter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>51</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="slider_super_fast">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Super Fast</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>140</y>
        <width>471</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
       <item row="2" column="1">
        <widget class="QLabel" name="samplerate_value">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="hop_s_label">
         <property name="text">
          <string>Number of frames between two consecutive runs:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="win_s_value">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="hop_s_value">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="win_s_label">
         <property name="text">
          <string>Length of FFT:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="samplerate_label">
         <property name="text">
          <string>Samplerate:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
